.jshintrc

```diff
@@ -27,17 +27,25 @@
 	"browser": true,
 
 	// Globals
-	"predef": [
-		"CompletenessTest",
-		"dataValues",
-		"globeCoordinate",
-		"jQuery",
-		"mediaWiki",
-		"QUnit",
-		"sinon",
-		"time",
-		"util",
-		"valueFormatters",
-		"valueParsers"
-	]
+	"globals": {
+		"CompletenessTest": false,
+		"dataValues": false,
+		"globeCoordinate": false,
+		"jQuery": false,
+		"QUnit": false,
+		"sinon": false,
+		"time": false,
+		"util": false,
+		"valueFormatters": false,
+		"valueParsers": false
+	},
+
+	"overrides": {
```

Comment:
Personally, I would prefer applying overrides with an inline comment in the file.